<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BloodNewWindows</class>
 <widget class="QWidget" name="BloodNewWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QtCharts::QChartView" name="graphicsView_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除曲线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartAndStop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动定时器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>79</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
